components/survey-from.tsx

```
@@ -0,0 +1,318 @@
+              "bekerja",
+              "melanjutkan pendidikan",
+              "wirausaha",
+              "belum bekerja",
+            ].map((section) => (
+              <div
+                className={cn(
+                  "flex items-center space-x-2 border-2 p-3 rounded-md cursor-pointer",
+                  selectedSection === section
+                    ? "border-green-500 text-green-500"
+                    : ""
+                )}>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)